config/wrapper.jinja2

```diff
@@ -0,0 +1,270 @@
+    </script>
+    <!-- end google tag manager -->
+
+    <header class="banner global" role="banner">
+      <nav role="navigation" class="nav-primary nav-right" id="nav">
+        <span id="main-navigation-link"><a href="#main-navigation">Jump to site nav</a></span>
+        <div class="logo">
+          <a href="/">
+            <img src="/static/img/maas-logo.png" alt="MAAS" />
+          </a>
+        </div>
+        <ul>
+          <li><a href="/get-started" {% if level_1 == 'get-started' %} class="active"{% endif %}>Get started</a></li>
+          <li><a href="/how-it-works" {% if level_1 == 'how-it-works' %} class="active"{% endif %}>How it works</a></li>
+          <li><a href="/tour" {% if level_1 == 'tour' %} class="active"{% endif %}>Tour</a></li>
+          <li><a href="/docs" {% if level_1 == 'docs' %} class="active"{% endif %}>Docs</a></li>
+        </ul>
+      </nav>
+      <div class="nav-toggle">
+        <a href="#nav" class="nav-toggle__link open"></a>
+        <a href="#" class="nav-toggle__link close"></a>
+      </div>
+    </header>
+
+    <div class="wrapper documentation__wrapper">
+      <nav class="documentation__nav" id="section-nav">
+        <section class="documentation__jump">
+          <a href="#">Back to top</a>
+        </section>
+
+        <section class="documentation__versions">
+          <h4>Versions</h4>
+
+          <ul>
+            <li><a class="active" href="/docs">2.0</a></li>
+            <li><a class="external" href="http://maas.ubuntu.com/docs1.9/">1.9</a></li>
+            <li><a class="external" href="http://maas.ubuntu.com/docs1.7/">1.7</a></li>
+            <li><a class="external" href="http://maas.ubuntu.com/docs1.5/">1.5</a></li>
+          </ul>
+        </section>
+
+        <ul>
+          {% for item in navigation %}
+          <li>
+            {% if item.location %}
+            <a href="{{ item.location | replace('.html', '') }}">{{ item.title }}</a>
+            {% else %}
+            <h4 class="header toggle-target">{{ item.title }}</h4>
+            {% endif %}
+
+            {% if item.children %}
+            <ul>
+              {% for child in item.children %}
+              <li class"section">
+                {% if child.location %}
+                <a class"header" href="{{ child.location | replace('.html', '') }}">{{ child.title }}</a>
+                {% else %}
+                {{ child.title }}
+                {% endif %}
+
+                {% if child.children %}
+                <ul>
+                  {% for grandchild in child.children %}
+                  <li><a href="{{ grandchild.location | replace('.html', '') }}">{{ grandchild.title }}</a></li>
+                  {% endfor %}
+                </ul>
+                {% endif %}
+              </li>
+              {% endfor %}
+            </ul>
+            {% endif %}
+          </li>
+          {% endfor %}
+        </ul>
+      </nav>
+
+      <script>
+        /**
+         * Polyfills:
```

Comment:
actually currently the documentation builder doesn't support template includes, it can only use a single template. We can improve this in the future, but not for this PR is that's okay.